packages/devtools-components/webpack.config.js

```diff
@@ -0,0 +1,160 @@
+const { isDevelopment, isFirefoxPanel, getValue, setConfig } = require("devtools-config");
+const NODE_ENV = process.env.NODE_ENV || "development";
+const TARGET = process.env.TARGET || "local";
+
+const defaultBabelPlugins = [
+  "transform-flow-strip-types"
+];
+
+module.exports = (webpackConfig = {}, envConfig) => {
+  setConfig(envConfig);
+
+  webpackConfig.context = path.resolve(__dirname, "src");
+  // webpackConfig.entry = "./index";
+  webpackConfig.devtool = "source-map";
+
+  webpackConfig.module = webpackConfig.module || {};
+  webpackConfig.module.rules = webpackConfig.module.rules || [];
+  webpackConfig.module.rules.push({
+    test: /\.json$/,
+    loader: "json-loader"
+  });
+  webpackConfig.module.rules.push({
+    test: /\.js$/,
+    exclude: request => {
+      // Some paths are excluded from Babel
+      let excludedPaths = ["fs", "node_modules"];
+      let excludedRe = new RegExp(`(${excludedPaths.join("|")})`);
+      let excluded = !!request.match(excludedRe);
+
+      if (webpackConfig.babelExcludes) {
+        // If the tool defines an additional exclude regexp for Babel.
+        excluded = excluded || !!request.match(webpackConfig.babelExcludes);
+      }
+      return excluded && !request.match(/node_modules(\/|\\)devtools-/);
+    },
+    loader: `babel-loader?${defaultBabelPlugins.map(
+      p => `plugins[]=${p}`
+    )}&ignore=src/lib`
+    // isJavaScriptLoader: true
+  });
+  webpackConfig.module.rules.push({
+    test: /\.svg$/,
+    loader: "svg-inline-loader"
+  });
+
+  webpackConfig.module.rules.push({
+    test: /\.properties$/,
+    loader: "raw-loader"
+  });
+
+  // Add resolveLoader for ./node_modules to fix issues when synlinked.
+  webpackConfig.resolveLoader = webpackConfig.resolveLoader || {};
+  webpackConfig.resolveLoader.modules = webpackConfig.resolveLoader.modules || [];
+  webpackConfig.resolveLoader.modules.push("node_modules");
+  webpackConfig.resolveLoader.modules.push(path.resolve(__dirname, "./node_modules"));
+
+  // Add a resolve alias for React if the target is UMD
+  if (webpackConfig.output && webpackConfig.output.libraryTarget === "umd") {
+    webpackConfig.resolve = webpackConfig.resolve || {};
+    webpackConfig.resolve.alias = webpackConfig.resolve.alias || {};
+    if (!webpackConfig.resolve.alias.react) {
+      webpackConfig.resolve.alias.react = "react/lib/ReactUMDEntry";
+    }
+  }
+
+  webpackConfig.node = { fs: "empty" };
+
+  webpackConfig.plugins = webpackConfig.plugins || [];
+  webpackConfig.plugins.push(
+    new webpack.DefinePlugin({
+      "process.env": {
+        NODE_ENV: JSON.stringify(NODE_ENV),
+        TARGET: JSON.stringify(TARGET)
+      },
+      DebuggerConfig: JSON.stringify(envConfig)
+    })
+  );
+
+  if (isDevelopment()) {
+    webpackConfig.module.rules.push({
+      test: /\.css$/,
+      use: [
+        { loader: "style-loader" },
+        { loader: "css-loader", options: { importLoaders: 1 } },
+        // {
```

Comment:
I copied the webpack config from the launchpad, which is not the right thing to do :)